bin/geographic_gatherer.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2014 - 2021 PyTroll developers
+#
+# Author(s): Martin Raspaud
+#            Panu Lahtinen
+#            Adam Dybbroe
+#
+#   Martin Raspaud <martin.raspaud@smhi.se>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Gather granule messages to send them in a bunch."""
```

Comment:
This should be update to mention 'geographic'

Reply:
Clarified the docstring.